tests/tools/kopiarunner/kopia_snapshotter_upgrade_test.go

```diff
@@ -0,0 +1,49 @@
+package kopiarunner
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestUpgradeFormatVersion(t *testing.T) {
+	baseDir, err := os.MkdirTemp("", t.Name())
+	require.NoError(t, err)
+
+	defer os.RemoveAll(baseDir)
+
+	repoDir, err := os.MkdirTemp(baseDir, "repo")
+	require.NoError(t, err)
+
+	_, err = os.MkdirTemp(baseDir, "source")
+	require.NoError(t, err)
```

Comment:
Use `t.TempDir()` it does automatic directory cleanup.

You can simplify this block by call `t.TempDir()` twice:

```suggestion
	repoDir := t.TempTemp()
	// Later when needed add => sourceDir := t.MkdirTemp()
```

Reply:
Done